File: java/src/HotelReview.java
public class HotelReview {
  private String hotel;
  private String member;
  private int score;

  public HotelReview(String hotel, String member, int score) {
      this.hotel = hotel;
      this.member = member;
      this.score = score;
  }

  public String getHotel() {
      return hotel;
  }

  public String getMember() {
      return member;
  }

  public int getScore() {
      return score;
  }

  public String toString() {
      return "Hotel: " + hotel + "; Member: " + member + "; Score: " + score;
  }
}

File: ruby/app/models/hotel_review.rb
class HotelReview < ActiveRecord::Base

  attr_accessible :hotel_id, :member_id, :score

  belongs_to :hotel
  belongs_to :member

  validates :hotel, presence: true
  validates :member, presence: true
  validates :score, inclusion: { in: 1..10 }

end
